src/rules.js

```diff
@@ -2687,5 +2687,72 @@ module.exports = [
         "arguments": [{
             "a": "auto"
         }]
+    },
+    /**
+    ==================================================================
+    SVG
+    ==================================================================
+    */
+    {
+        "type": "pattern",
+        "name": "fill",
+        "matcher": "Fill",
+        "allowParamToValue": true,
+        "styles": {
+            "fill": "$0"
+        },
+        "arguments": [colors]
+    },
+    {
+        "type": "pattern",
+        "name": "stroke",
+        "matcher": "Stk",
+        "allowParamToValue": true,
```

Comment:
Is this one limited only to the values in `colors`?  If so, I think `allowParamToValue` should be false